examples/shader/compute_shader_game_of_life_pipelined.rs

```diff
@@ -0,0 +1,215 @@
+            source: ShaderSource::Wgsl(shader_source.into()),
+        });
+
+        let texture_bind_group_layout =
+            render_device.create_bind_group_layout(&BindGroupLayoutDescriptor {
+                label: None,
+                entries: &[BindGroupLayoutEntry {
+                    binding: 0,
+                    visibility: ShaderStages::COMPUTE,
+                    ty: BindingType::StorageTexture {
+                        access: StorageTextureAccess::ReadWrite,
+                        format: TextureFormat::Rgba8Unorm,
+                        view_dimension: TextureViewDimension::D2,
+                    },
+                    count: None,
+                }],
+            });
+
+        let pipeline_layout = render_device.create_pipeline_layout(&PipelineLayoutDescriptor {
+            label: None,
+            bind_group_layouts: &[&texture_bind_group_layout],
+            push_constant_ranges: &[],
+        });
+        let init_pipeline = render_device.create_compute_pipeline(&ComputePipelineDescriptor {
+            label: None,
+            layout: Some(&pipeline_layout),
+            module: &shader,
+            entry_point: "init",
+        });
+        let sim_pipeline = render_device.create_compute_pipeline(&ComputePipelineDescriptor {
+            label: None,
+            layout: Some(&pipeline_layout),
+            module: &shader,
+            entry_point: "update",
+        });
+
+        GameOfLifePipeline {
+            sim_pipeline,
+            init_pipeline,
+            texture_bind_group_layout,
+        }
+    }
+}
+
+enum Initialized {
+    Default,
+    No,
+    Yes,
+}
+
+struct DispatchGameOfLife {
+    initialized: Initialized,
+}
+impl Default for DispatchGameOfLife {
+    fn default() -> Self {
+        Self {
+            initialized: Initialized::Default,
+        }
+    }
+}
+impl render_graph::Node for DispatchGameOfLife {
+    fn update(&mut self, _world: &mut World) {
+        match self.initialized {
+            Initialized::Default => self.initialized = Initialized::No,
+            Initialized::No => self.initialized = Initialized::Yes,
+            Initialized::Yes => {}
+        }
+    }
+
+    fn run(
+        &self,
+        _graph: &mut render_graph::RenderGraphContext,
+        render_context: &mut bevy::render2::renderer::RenderContext,
+        world: &World,
+    ) -> Result<(), render_graph::NodeRunError> {
+        let pipeline = world.get_resource::<GameOfLifePipeline>().unwrap();
+        let texture_bind_group = &world.get_resource::<GameOfLifeImageBindGroup>().unwrap().0;
+
+        let mut pass = render_context
+            .command_encoder
+            .begin_compute_pass(&ComputePassDescriptor::default());
+
+        if let Initialized::No = self.initialized {
+            pass.set_pipeline(&pipeline.init_pipeline);
+            pass.set_bind_group(0, texture_bind_group, &[]);
+            pass.dispatch(SIZE.0 / WORKGROUP_SIZE, SIZE.0 / WORKGROUP_SIZE, 1);
```

Comment:
Shouldn't this be `pass.dispatch(SIZE.0 / WORKGROUP_SIZE, SIZE.1 / WORKGROUP_SIZE, 1);` (SIZE.1 instead of SIZE.0 in y coordinate)?

Reply:
Correct, fixed it now